SECURITY.md 

```diff
@@ -0,0 +1,39 @@
+# Security Policy
+
+Security is very important for Fetch.ai and its community. This document outlines security procedures and general policies for the `uagents` project.
+
+## Supported Versions
+
+[Here](https://github.com/fetchai/uAgents/releases) you can find which versions of `uagents` are currently being supported with security updates.
+
+## Reporting a Vulnerability
```

Comment:
Please replace with the `How to Report` section [here](https://github.com/fetchai/ecosystem-tasks/blob/main/bug-bounty.md#how-to-report) removing any references to bug bounty.